### DISCUSSION TITLE
Problems with 7" Displays

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1463] [Bug Reports]
@rern Thanks for the local display brightness fix!

The GUI is not scaled correctly on both a local 7" touch screen and on a 7" Android tablet. In landscape mode the bottom bar has about a third missing from the lower edge. Also, if a screen-off time of 10 is selected the disable option is then hidden on the touch screen drop-down.

**Comment by ceejayemm** [comment]
@R101G 
I also reported this (as #1429) but, for me at least, it seems to be fixed with 20231020.

Chris

**Reply by R101G** [reply]
@ceejayemm 
I'm on 20231022 now, and it's still there.

EDIT: The scaling affects an Android tablet as well as the local display, so I am not sure if it is the same problem as you had.

**Reply by ceejayemm** [reply]
Seems I spoke too soon, Zoom seems to work but not anything else. 

Chris

**Reply by ceejayemm** [reply]
Further, it seems changes to the 'Browser on RPi' settings are only taking place on a reboot rather that straightaway as used to be the case.

Chris

**Comment by R101G** [comment]
Looking at the Android app, the GUI with top and bottom bars does not appear to resize correctly in landscape mode, but it does in portrait mode. My phone has the same problem, so it's not just 7" displays.

**Reply by rern** [reply]
Please try the UPDATE branch.

**Reply by R101G** [reply]
Thanks @rern 
That seems to have fixed it.
